src/steps/resource-manager/compute/constants.ts

```diff
@@ -75,6 +79,11 @@ export const entities = {
     _type: VIRTUAL_MACHINE_ENTITY_TYPE,
     _class: VIRTUAL_MACHINE_ENTITY_CLASS,
   },
+  VIRTUAL_MACHINE_SCALE_SET: {
+    resourceName: '[RM] Virtual Machine Scale Set',
+    _type: VIRTUAL_MACHINE_SCALE_SET_ENTITY_TYPE,
+    _class: VIRTUAL_MACHINE_SCALE_SET_ENTITY_CLASS,
```

Comment:
I recommend setting `_type` and `_class` here, to avoid temptation to import `VIRTUAL_MACHINE_SCALE_SET_ENTITY_TYPE` in other files.

You haven't done this, but this becomes easier if `_type` and `_class` have their own exported variables.

```diff
  return createIntegrationEntity({
    entityData: {
      source: data,
      assign: {
        _key: data.id!,
-       _type: VIRTUAL_MACHINE_SCALE_SET_ENTITY_TYPE,
+       _type: entities.VIRTUAL_MACHINE_SCALE_SET._type,
-       _class: VIRTUAL_MACHINE_SCALE_SET_ENTITY_CLASS,
+       _class: entities.VIRTUAL_MACHINE_SCALE_SET._class,
```

```suggestion
    _type:'azure_vm_scale_set',
    _class: ['Deployment', 'Group'],
```